lib/restate-constructs/register-service-handler/index.ts

```diff
@@ -194,8 +194,9 @@ export const handler: Handler<CloudFormationCustomResourceEvent, void> = async f
         if (props.servicePath && !response.services.find((s) => s.name === props.servicePath)) {
           failureReason =
             `"Registration succeeded, but none the services names in the deployment matched the specified name. " + 
```

Comment:
Had another look at that line for some reason and noticed that the <code>`"</code> also looks suspicious.

Reply:
I'm 99% certain this is IDEA sometimes being too clever with copy-paste - thanks for spotting this!